vms/vms/settings.py

```diff
@@ -61,8 +61,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'myuser',
-        'PASSWORD': 'mypassword',
+        'USER': 'vmsadmin',
+        'PASSWORD': '0xdeadbeef',
         'HOST': 'localhost',
```

Comment:
> If its intentional to leave it that way, then its fine.

Cool. I thought what you did, does affect the syncdb issue, somehow that I'm not aware of. That deadbeef made me curious. :smile: 

> Btw, I attended the Codeathon @ Grace Hopper 2015 last week. 

That's great. Your first PR does fixes something great. 

> Still not
> familiar with the standards followed by this project.

No issues. You will learn as you contribute more. Feel free to open issues if you don't find something in documentation. If a beginner is not easily able to do it, there is some fault in our documentation. You can also fix such issues you find.